src/js/components/GroupNav/GroupNav.js

```diff
@@ -1,23 +1,30 @@
 import React, { Component } from "react";
 import '../../../scss/components/GroupNav/GroupNav.scss';
+import GroupNavDMButtons from "../GroupNavDMButtons/GroupNavDMButtons";
 
 class GroupNav extends Component {
   render() {
     return(
       <div className="nav-column nav-column-message">
         <div className="nav-group nav-group-top">
-          <div className="nav-item-start">
-            <button className="btn btn-start">Find or start a conversation</button>
+          <div className="nav-item nav-item-start">
+            <button className="nav-item-start-content">Find or start a conversation</button>
```

Comment:
This confuses me a little, you can `nav-item`s that contain `nav-items-*`? It makes more sense to me to have `nav-item`s that contain `btn`s